PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue

```diff
@@ -65,121 +57,106 @@
         >
           <template v-slot:activator="{ on }">
             <v-text-field
-				v-model="householdDate"
-				:rules="birthdateRules"
+              v-model="birthDate"
```

Comment:
This one as well?

Reply:
"birthDate" makes more sense than "householdDate" for the Date of Birth field